packages/clients/meldemichel/src/plugins/AfmButton/AfmButton.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-btn
+    v-if="afmUrl"
+    :target="mapStateReady ? '_blank' : ''"
+    color="primary"
+    class="meldemichel-afm-button"
+    :href="mapStateReady ? afmUrl : '#'"
+    large
```

Comment:
```suggestion
```
Looks better without it in my opninion

Reply:
Our opinions differ in this decision, albeit I did not consider which version is more aesthetic. I fathomed the main interaction button (that allows proceeding in the reporting process) demands a lottle more weight and visibility. After all, it also helps with the procedure by toasting what is to be done in the client.

Reply:
If one is not using `large`, it still seems reasonably important. With `large` it seems almost obstructive to me. As we don't have a requirement though -> 🎩